scripts/build/README.md

```diff
@@ -1,5 +1,9 @@
 # Prettier Build Script
 
+## Requirements
+
+- Node.js version `>= 14.18`.
```

Comment:
It actually works with a slightly older version, but this is the most current minor version.